vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,90 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip3 install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python3 update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
```

Comment:
Can you say more about the structure of `updateInfo`? I'm not real clear what it should contain. Maybe an example use case.